lib/node_modules/@stdlib/ndarray/flatten/README.md

```diff
@@ -0,0 +1,127 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# flatten
+
+> Return a flattened copy of an input [`ndarray`][@stdlib/ndarray/ctor].
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var flatten = require( '@stdlib/ndarray/flatten' );
+```
+
+#### flatten( x, depth )
```

Comment:
Also, similar to NumPy (ref: https://numpy.org/doc/stable/reference/generated/numpy.ravel.html#numpy-ravel), we should support specifying a "read" order.

```
flatten( x[, opts] )
```

with options

- **depth**: default infinity.
- **order**: read order. If `row-major`, always flatten an input array in lexicographic order. If `column-major`, always flatten in colexicographic order. If `same`, flatten in the order of the input array (i.e., equivalent to `order = getOrder( x )`). If `any`, flatten in colexicographic order if input array **data** is column-major contiguous; otherwise, flatten in lexicographic order (i.e., flatten according to how the data is arranged in memory, which may not correspond to the stated array order).

Reply:
Additional references:

- Order options: https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/ndarray/array
- "colexicographic" terminology: https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/array/base/flatten